File: addons/confirma/src/types/TestResult.cs
namespace Confirma.Types;

public struct TestResult
{
	public uint TotalTests { get; set; } = 0;
	public uint TestsPassed { get; set; } = 0;
	public uint TestsFailed { get; set; } = 0;
	public uint TestsIgnored { get; set; } = 0;
	public double TotalTime { get; set; } = 0;
	public uint Warnings { get; set; } = 0;

	public TestResult() { }
}

File: e2e/types/test_result.ts
import type { TestLog } from "./test_log";

export interface TestResult {
  TotalTests: number;
  TestsPassed: number;
  TestsFailed: number;
  TestsIgnored: number;
  TotalOrphans: number;
  TotalClasses: number;
  TotalTime: number;
  Warnings: number;
  TestLogs: TestLog[];
}
